examples/asr/speech_multitask/speech_to_text_aed.py

```diff
@@ -55,6 +55,7 @@
 from omegaconf import OmegaConf
 
 from nemo.collections.asr.models import EncDecMultiTaskModel
+from nemo.collections.common.tokenizers import CanaryTokenizer
```

Comment:
Do not special case the Canary tokenizer inside of a generic example script. Use hydra class import resolution

Reply:
We need the class method, not an actual instance of the tokenizer. Won't hydra import simply try to instantiate a class?

Also, the changes to CanaryTokenizer make it more general purpose now. (it's actually more of an `speech_prompt_tokenizer`)